Applications/AlgorithmPlaza.qmd

```diff
@@ -50,46 +54,48 @@ The table below provides an overview of the different maturity levels that are a
 
 For more information on dealing with the services, please refer to the [Manage your services guide](PlazaDetails/ManageService.qmd).
 
+Once familiar with the concepts used in the openEO Algorithm Plaza, individuals can begin to explore the platform.
 
 ## Interested in using or publishing services?
 
-As a user, you have the option to either use an existing service or publish your own algorithms. Here, we have provided a list of important steps to consider when using or sharing a service. It's worth noting that this list is not exhaustive and additional steps may be required based on your specific needs.
+Users can either use an existing service or publish their algorithms. Here, a list of important steps to consider when using or sharing a service is provided. It's worth noting that this list is not specific, and additional steps may be required based on specific needs.
```

Comment:
Here? I do not see a link? Maybe use below in stead. 

Below a non-extensive list is provided of considerations to take when sharing a service. 

 Be writing non-extensive you can leave out the second sentence